lib/webdrivers/chromedriver.rb

```diff
@@ -16,7 +16,13 @@ def current_version
       end
 
       def latest_version
-        raise StandardError, 'Can not reach site' unless site_available?
+        unless site_available?
+          cur_ver = current_version
```

Comment:
Not as big of a deal since they aren't network calls, but some paths through this code are also calling `#current_version` multiple times in a single update. Let's memoize it and then set it back to `nil` in the `#remove` method.